src/rgw/rgw_trim_bilog.cc

```diff
@@ -451,6 +453,8 @@ class BucketTrimInstanceCR : public RGWCoroutine {
       http(http), observer(observer),
       bucket_instance(bucket_instance),
       zone_id(store->svc()->zone->get_zone().id),
+#warning Get this from somewhere?
```

Comment:
what is the implication for the code as below?

Reply:
it's a placeholder for a future piece to fill in. There's something similar (a comment rather than a warning) I found in the list code when looking for where to get the right generation.

Reply:
when we query the bucket sync status from peer zones, we'll need to get their current generation along with the markers. it looks like we'll need to extend `RGWOp_BILog_Status` to include the `rgw_bucket_sync_status::incremental_gen` as well

the generation we trim will be the min(peer gens). and if we still have log gens older than that, we can delete their index objects